func_tests/test_client.py

```diff
@@ -1,6 +1,38 @@
 from func_tests.vinyldns_context import vinyldns_test_context
+from vinyldns.record import RecordSet, RecordType, AData
+
```

Comment:
I expanded on the existing func tests a little to at least hit zones, groups, record sets.

This is not an exhaustive set of tests.  The most important is create/delete record set and create/delete zone which most clients will use.